.github/workflows/release.yml

```diff
@@ -13,12 +13,32 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Build and push image
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: dexidp/dex
-          tags: latest
-          tag_with_ref: true
-          add_git_labels: true
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+          version: latest
+          driver-opts: image=moby/buildkit:master
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and Push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          push: true
+          tags: |
+            dexidp/dex:latest
```

Comment:
We have to push a versioned image as well, which is specified by the `git tag` of the release.

Reply:
Good catch, I've added that logic now to also push the image with the version using the same script that determines the tag version to publish currently.

Reply:
Please also add an equivalent to `add_git_labels`. See the action's README for details.

Reply:
Added